lib/livebook/system_resources.ex

```diff
@@ -66,11 +66,21 @@ defmodule Livebook.SystemResources do
 
   defp measure() do
     memory_data = :memsup.get_system_memory_data()
-    memory = %{total: memory_data[:total_memory], free: memory_data[:free_memory]}
+    free_memory = measure_free_memory(Enum.into(memory_data, %{}))
```

Comment:
```suggestion
    free_memory = measure_free_memory(Map.new(memory_data))
```

Reply:
Also, maybe `determine_free_memory` or just `free_memory` since it doesn't measure? :D